.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Release Gist Apple
+run-name: Release gist-apple ${{ github.event.inputs.release_version }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release version'
+        required: true
+
+jobs:
+  release:
+    runs-on: macos-latest
+    env:
+      GIST_COCOAPODS_TRUNK_TOKEN: ${{ secrets.GIST_COCOAPODS_TRUNK_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: 'develop'
+          fetch-depth: 0
+      - name: Update podspec
+        shell: bash
+        run: |
+          RELEASE_VERSION="${{ github.event.inputs.release_version }}"
+          echo "RELEASE_VERSION=$RELEASE_VERSION" >> $GITHUB_ENV
+          echo "RELEASE_VERSION=$RELEASE_VERSION"
+
+          sed -i -E "s/s\.version \= '.*'/s\.version = '$RELEASE_VERSION'/" Gist.podspec
+          rm Gist.podspec-E || true
+      - name: Merge to develop
+        shell: bash
+        run: |
+          git config user.name "ami-ci"
+          git config user.email "<>"
+          git checkout develop
+          git add .
+          set +e  # Grep succeeds with nonzero exit codes to show results.
+          git status | grep modified
+          if [ $? -eq 0 ]
+          then
+              set -e
+              git commit -am "New version of gist-web"
+              git push
+          else
+              set -e
+              echo "No changes since last run"
+          fi
+      - name: Release to master
+        shell: bash
+        run: |
+          echo "Release $RELEASE_VERSION"
+          git remote update
+          git checkout release/$RELEASE_VERSION 2>/dev/null || git checkout -b release/$RELEASE_VERSION origin/develop
+          git push origin release/$RELEASE_VERSION
```

Comment:
Like on web, I think we can just bypass the whole release branch thing since it's not serving any real purpose here.

The real-life use case of a release branch would be to "cut out" a release and for a period of time test and apply fixes before it gets merged to develop and master. In this case its being opened and closed right away.